localization/yabloc/yabloc_particle_filter/src/camera_corrector/camera_particle_corrector_core.cpp

```diff
@@ -187,8 +190,8 @@ void CameraParticleCorrector::on_line_segments(const PointCloud2 & line_segments
 
   // DEBUG: just visualization
   {
-    Pose mean_pose = get_mean_pose(weighted_particles);
-    Sophus::SE3f transform = common::pose_to_se3(mean_pose);
+    Pose tmp_mean_pose = get_mean_pose(weighted_particles);
+    Sophus::SE3f transform = common::pose_to_se3(tmp_mean_pose);
```

Comment:
this is for avoiding variable shadowing. I didn't come up for a good name...

Reply:
You can avoid the declaration of `tmp_mean_pose` :+1:
```suggestion
    Sophus::SE3f transform = common::pose_to_se3(get_mean_pose(weighted_particles););
```

Reply:
@a-maumau Sorry. My suggestion contained an invalid `;`. Please correct it. :bow:

Reply:
I fixed it at 31e2994 .